frontend/app/src/store/balances/blockchain-accounts.ts

```diff
@@ -458,13 +460,21 @@ export const useBlockchainAccountsStore = defineStore(
         const { refreshPrices, fetchNfBalances } = useBalancesStore();
         startPromise(fetchNfBalances());
 
-        if (blockchain === Blockchain.ETH) {
+        (async () => {
```

Comment:
Don't we have an independent action about fetching balances? this seems like something that would be in it's own action

Reply:
Move to a function but not own action. I don't find any usage in other place, check again is it okay or not